apps/backend/src/evaluations/evaluations.controller.ts

```diff
@@ -85,13 +87,18 @@ export class EvaluationsController {
       evaluationTags: createEvaluationDto.evaluationTags || [],
       public: createEvaluationDto.public
     };
+    // Do not include userId on the DTO so we can set it automatically to the uploader's id.
+    const createdDto = await this.evaluationsService.create(
+      updatedEvaluationDto,
+      serializedDta,
+      request.user.id
+    );
     return new EvaluationDto(
-      // Do not include userId on the DTO so we can set it automatically to the uploader's id.
-      await this.evaluationsService.create(
-        updatedEvaluationDto,
-        serializedDta,
-        request.user.id
-      )
+      createdDto,
+      false,
```

Comment:
If the user just created an evaluation then shouldn't editable be true since they are the creator?